app/src/main/java/com/partymusicq/android/pojo/SongQueue.kt

```diff
@@ -0,0 +1,62 @@
+package com.partymusicq.android.pojo
+
+class SongQueue(private var songQueue: ArrayList<Song>) {
```

Comment:
I question the need for this object at all. I think that all of the functions that you have is already implemented if you were to use an Arraylist or Hashset. 
I think a Hashset the best.

Reply:
there definitely isn't anything built in that can handle updating the position of songs on the song object

Reply:
Actually coming to think about this. We don't want to do it this way anyway. I think that we would have to sit down and talk about this. because the queue should always be live updated to the database.